tests/ops/functions/conftest.py

```diff
@@ -157,6 +157,7 @@ def _trotterize_qfunc_dummy(time, theta, phi, wires, flip=False):
     PowOperation,
     PowObs,
     qml.StatePrep,
+    qml.FromBloq
```

Comment:
For now I've put qml.FromBloq as one of those instances that aren't supposed to be tested in `test_assert_valid.py` because it requires an optional dependency. Seems like it would make sense for this to be excluded.